susemanager/src/mgr_bootstrap_data.py

```diff
@@ -142,6 +142,44 @@
     "venv-salt-minion",
 ]
 
+PKGLISTDEBIAN13 = [
+    # gnupg dependencies
```

Comment:
I tested this on a container, so this list will cover all cases for absolutely any VM we may encounter, no matter how minimal the installation is.

I am also using a single list of packages for Debian and Raspberry Pi OS. I checked that on 12 we have two lists, but from what I could see  they were the same.

Being Raspberry Pi OS a clone of Debian (for `arm64` is not even a rebuild as it uses the repos from Debian), it should be OK.